src/entity/dataset_title.ts

```diff
@@ -0,0 +1,28 @@
+import { Entity, PrimaryColumn, Column, BaseEntity, ManyToOne, JoinColumn } from 'typeorm';
+
+// eslint-disable-next-line import/no-cycle
+import { Dataset } from './dataset';
+
+@Entity({ name: 'dataset_title' })
+export class DatasetTitle extends BaseEntity {
+    @PrimaryColumn({ name: 'dataset_id' })
+    datasetId: string;
+
+    @ManyToOne(() => Dataset, (dataset) => dataset.description, { onDelete: 'CASCADE' })
+    @JoinColumn({ name: 'dataset_id' })
+    dataset: Dataset;
+
+    @Column({ nullable: false })
+    title: string;
+
+    @PrimaryColumn({ name: 'language_code' })
+    languageCode: string;
+
+    public static datasetTitleFromString(dataset: Dataset, title: string, language: string): DatasetTitle {
+        const datasetTitle = new DatasetTitle();
+        datasetTitle.dataset = dataset;
+        datasetTitle.title = title;
+        datasetTitle.languageCode = language;
+        return datasetTitle;
+    }
+}
```

Comment:
So I wanted to be clear that a dataset would have multiple titltes and descriptions in multiple languagues all of which hang off the main dataset object.  Because a dataset might not have a title or descriptions especially very early in the use journey I wanted to make it very easy to understand.  Additionally I still wanted to provide a title to the frontend even if we only had a title in a single language.  Much of the metadata relates to codes, licenses and things which don't have a translation.  Much of that metadata I expect to exist on the main dataset table.  If we end up with a lot more bits of meta data in multiple languages I'm happy to revisit this design to reduce the boiler plate.